tests/providers/docker/operators/test_docker.py

```diff
@@ -86,6 +87,7 @@ def test_execute(self):
             host_config=self.client_mock.create_host_config.return_value,
             image='ubuntu:latest',
             user=None,
+            entrypoint=['sh', '-c'],
```

Comment:
Are there tests *without* `entrypoint` now?

Reply:
there is [one test](https://github.com/jsemric/airflow/blob/entrypoint-support-in-docker-operator/tests/providers/docker/operators/test_docker.py#L229) testing `xcom` without `entrypoint`

Reply:
Cool, thanks!